### PR TITLE
Office hours, we are open

### PR DESCRIPTION
**Pull request by kpodemski**
Communication about office hours.

### CODE REVIEW

**Review by nicosomb — APPROVED** [review]

File: news/_posts/2022-11-17-office-hours.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+subtitle: "Come discuss about the project with its members!"
+date:  2022-11-17
+authors: [ kpodemski ]
+icon: icon-chat
+tags: [community]
+---
+
+A few weeks ago, we started a new initiative: PrestaShop Project Office Hours.
+
+The concept is simple: Once a week, project members make themselves available to the community to discuss topics directly related to the PrestaShop open source project.
```
Comment (coffrep) [inline news/_posts/2022-11-17-office-hours.md]:
```suggestion
The concept is simple: once a week, project members make themselves available to the community to discuss topics directly related to the PrestaShop open source project.
```

File: news/_posts/2022-11-17-office-hours.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+So far, we have held three meetings which were a chance for us to discuss a number of topics:
+- project structure, processes
```
Comment (coffrep) [inline news/_posts/2022-11-17-office-hours.md]:
```suggestion
- project structure and processes
```

File: news/_posts/2022-11-17-office-hours.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+## We are open!
+
+Would you like to discuss a Pull Request you are working on? Or about the future of the project? Do you just want to listen to the discussion? If so, joining the meeting will be a great idea.
+
+## When and where?
+
+Feel free to join us on [this Google Meet link](https://meet.google.com/fer-hcgk-wti).
+Office Hours are held *every Tuesday* (except days that are bank holidays in most European countries) from 11:00 AM to 12 PM CET.
+
+If you want to suggest topics for discussion, please [fill out this form](https://forms.gle/2r7jPp5WxhFSUQme8). (it is not required, though!)
```
Comment (coffrep) [inline news/_posts/2022-11-17-office-hours.md]:
```suggestion
If you want to suggest topics for discussion, please [fill out this form](https://forms.gle/2r7jPp5WxhFSUQme8) (it is not required, though!).
```

File: news/_posts/2022-11-17-office-hours.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+tags: [community]
+---
+
+A few weeks ago, we started a new initiative: PrestaShop Project Office Hours.
+
+The concept is simple: Once a week, project members make themselves available to the community to discuss topics directly related to the PrestaShop open source project.
+
+So far, we have held three meetings which were a chance for us to discuss a number of topics:
+- project structure, processes
+- we helped finish and merge some PRs from the community
```
Comment (coffrep) [inline news/_posts/2022-11-17-office-hours.md]:
```suggestion
- helping finish and merge some PRs from the community
```

File: news/_posts/2022-11-17-office-hours.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+- multi-store features
+- the future of core architecture
+
+And we hope to discuss more during the upcoming meetings.
+
+## We are open!
+
+Would you like to discuss a Pull Request you are working on? Or about the future of the project? Do you just want to listen to the discussion? If so, joining the meeting will be a great idea.
```
Comment (coffrep) [inline news/_posts/2022-11-17-office-hours.md]:
```suggestion
Would you like to discuss a Pull Request you are working on? Or the future of the project? Do you just want to listen to the discussion? If so, joining the meeting will be a great idea.
```

**Review by coffrep — APPROVED** [review]
I've added a few suggestions and approving the post

**Pull request merged**